docs/snippets/vue/form-story-component-with-play-function.3.js.mdx

```diff
@@ -0,0 +1,33 @@
+```js
+// Form.stories.js | Form.stories.jsx | Form.stories.ts | Form.stories.tsx
+
+import userEvent, { expect, screen } from '@storybook/addon-interactions';
+
+import LoginForm from './LoginForm.vue';
+
+export default {
+  component: LoginForm,
+  //👇 Marking the onSubmit with this configuration will log the event in the Actions panel
+  argTypes: { onSubmit: { action: true } },
+};
+
+export const FilledForm = {
+  play: async ({ args }) => {
```

Comment:
Maybe not necessarily in this PR, but I believe things changed in regards to the play function, where to make sure stories will work on docs mode you should query elements from a `canvasElement` (example):

```js
play: async ({ args, canvasElement }) => {
  await userEvent.click(within(canvasElement).getByRole('button'));
}
```

Reply:
cc @ghengeveld

Reply:
@yannbf thanks for the feedback and help on this. I'll defer to @ghengeveld for a bit more context and should the example needs some tweaks I'll update it accordingly.